docs/content/docs/reference/hasura-auth/configuration.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Configuration
+---
+
+## Email configuration
+
+It is possible to enable the following features in configuring an external SMTP email service:
+
+- Sign up
+- Password reset
+- Email change
+- Passwordless with emails
+
+### SMTP settings
+
+```
```

Comment:
Could you please add `bash` to this section? Otherwise a `$` symbol would be prepended to the text and the text block wouldn't expand to the full available width.